### PR TITLE
Task-50569: Resume the possibility to center the text on CkEditor

### PR DESCRIPTION
**Pull request by ahmadmido**

### CODE REVIEW

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue [hunk 1]
```diff
@@ -538,16 +538,11 @@ export default {
...
-            const attachMediaButton = document.getElementById('cke_18');
-            const attachFileButton = document.getElementById('cke_22');
             numerotationGroupButton.style.borderRight = 'none';
-            attachMediaButton.style.display = 'none';
-            attachFileButton.style.display = 'none';
             const spanBadge = document.createElement('span');
             spanBadge.setAttribute('class','badge');
             spanBadge.setAttribute('id','badge');
             spanBadge.innerHTML = '0';
-            attachFileButton.appendChild(spanBadge);
```
Comment (azayati) [inline webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue]: you should adapt them to the new ids, cc @hassenelkamel-exo 
To be verified and tested also the ids here https://github.com/exoplatform/news/pull/323/files#diff-22f9ace050d5a5d8e0e892116084dea8d84f4658621d15a49d7b66596bc161e3L982-L1009

**Review by azayati — CHANGES_REQUESTED** [review]
see review

File: webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue [hunk 2]
```diff
@@ -537,17 +538,19 @@ export default {
...
+              const numerotationGroupButton = document.getElementById('cke_14');
+              attachMediaButton.style.display = 'none';
+              attachFileButton.style.display = 'none';
+              attachFileButton.appendChild(spanBadge);
+              numerotationGroupButton.style.borderRight = 'none';
+            }
             spanBadge.setAttribute('class','badge');
             spanBadge.setAttribute('id','badge');
             spanBadge.innerHTML = '0';
-            attachFileButton.appendChild(spanBadge);
```
Comment (hassenelkamel-exo) [inline webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue:550]: you should put it in the code block of mobile

**Review by azayati — APPROVED** [review]
LGTM

**Review by hassenelkamel-exo — APPROVED** [review]
LGTM

**Pull request merged**